schemas/blocks/question.json

```diff
@@ -22,6 +22,26 @@
       "question_variants": {
         "$ref": "definitions.json#/question_variants"
       },
+      "list_summary": {
```

Comment:
Should we add this to content blocks as well?

Reply:
I don't know, but we don't have a requirement for this right now. We'd need to make changes to display it in the open PR on runner, which would require checking for the key in the content handler there. Seems overkill for now.